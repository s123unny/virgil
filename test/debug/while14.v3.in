b ../core/while14.v3 9
run
bt
info l
c
q
