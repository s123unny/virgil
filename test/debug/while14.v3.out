breakpoint|../core/while14.v3 9|0
stopOnBreakpoint
bt|while14.main|../core/while14.v3|9
variable|0|this|while14|while14|false
variable|1|arg|0|int|false
variable|2|i|10|int|false
end
result|0
